code/modules/uplink/uplink_items.dm

```diff
@@ -1707,6 +1707,13 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	cost = 2
 	restricted_species = list("plasmaman")
 
+/datum/uplink_item/race_restricted/syndie_discount_supersoldier_juice
+	name = "Syndicate Supersoldier Toxin"
+	desc = "A reverse engineered Nanotrasen supersoldier injector. Not as powerful as the Nanotrasen version but will still make you hella robust."
+	item = /obj/item/reagent_containers/hypospray/combat/supersoldier_syndie
+	cost = 20
```

Comment:
haha it being 20 tc totally makes it balanced right guys

Reply:
I'ill give it a pop limit

Reply:
please tell me if anything else has to be nerfed